### DISCUSSION TITLE
RTL Support

### DISCUSSION THREAD

**Discussion by movahhedi** [markdoc/markdoc#527] [Q&A] [answered]
Does Markdoc support RTL Languages? Arabic/Persian/Hebrew.

Individual isolated BiDi text block would be cool.

**Comment by mfix-stripe** [comment, marked as answer]
Hi @movahhedi — Markdoc is agnostic to such capabilities and you can easily implement the functionality you desire with custom Markdoc tags, or even just CSS.
